IPython/utils/warn.py

```diff
@@ -11,7 +11,7 @@
 import sys
 import warnings
 
-warnings.warn("The module IPython.utils.warn is deprecated, use the standard warnings module instead", DeprecationWarning)
+warnings.warn("The module IPython.utils.warn is deprecated since IPython 4.0 and will be remove in 6.0, use the standard warnings module instead", DeprecationWarning)
```

Comment:
Adding "since 4.0" is great, but I don't think we should schedule removal. I don't think deprecated code should be removed until:

1. it's been a really long time since ~everybody actually stopped using it (in years, not just releases), or
2. maintaining its existence causes a real problem